edflow/eval/pipeline.py

```diff
@@ -346,20 +348,17 @@ def after_epoch(self, epoch):
             kwargs = cb_kwargs.get(n, {})
             cb(self.root, self.data_in, data_out, self.config, **kwargs)
 
-    def save_csv(self):
+    def save_meta(self):
         """ """
-        csv_path = os.path.join(self.root, "model_output.csv")
 
-        if self.data_frame is not None:
-            self.data_frame = self.data_frame.sort_index()
-            self.data_frame.to_csv(csv_path, index=False)
+        if not hasattr(self, "exception_occured"):
+            had_exception = f"    .. note ::\n\n        No exception encountered during creation.\n\n"
```

Comment:
Do we want this note in case no exception happened? I think the non-exceptional case requires no mentioning.

Reply:
I would argue that this kind of documentation should be done to simply be sure everything went alright. It does not really cost anything and does not produce unnecessary log outputs. This information will be written to the `meta.yaml` of the created dataset, thus allowing anyone who wants to use this data to explicitly know if everything is created.